### ISSUE TITLE
请问delta的meshgrid那里是RAFT作者写错了是吗

### ISSUE THREAD

**Issue by alrisgn** [ispc-lab/TMA#9]
RAFT作者在corr.py里写的是
delta = torch.stack(torch.meshgrid(dy, dx), axis=-1)
你的代码里改成了
delta = torch.stack(torch.meshgrid(dx, dy, indexing="xy"), axis=-1)
我感觉你是对的，但我不敢判断。

**Comment by HTLeoo** [comment]
pytorch有一个版本更新了torch.meshgrid的语法，indexing参数有'xy'和‘ij’两种表示，可以去看看[torch.meshgrid](https://docs.pytorch.org/docs/stable/generated/torch.meshgrid.html)；目前默认应该还是‘ij’，所以RAFT这个写法肯定没问题。我习惯xy这个表示所以是dx在前，dy在后。

**Comment by alrisgn** [comment]
因为ij是行索引在前，xy是列索引在前，所以拿ij模式生成的索引网格当横、纵坐标的delta用，会导致这个窗口实际上是取完互相关值之后又沿主对角线翻转了一回，对吗？不过后续也都是拉成一维喂给网络，只要顺序是固定的，翻没翻转，看上去对训练也不会有啥影响，对吗？可能我有点过分较真了，多谢前辈回复。
